### PR TITLE
fix(deploy): remove resource limits

### PR DESCRIPTION
**Pull request by ecordell**
**Description of the change:**
Removes resource limits on OLM pods

**Motivation for the change:**

OLM resource usages scales with the size of a cluster. Having limits means that OLM can be killed by the scheduler on large clusters even though it is behaving normally.

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: deploy/chart/values.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-installType: upstream
```
Comment (awgreene) [inline deploy/chart/values.yaml]: Why is this removed?

Comment (ecordell) [reply deploy/chart/values.yaml]: This was duplicated - the same key/value is down below
